apps/server/src/modules/task-card/uc/task-card.uc.spec.ts

```diff
@@ -258,6 +258,49 @@ describe('TaskCardUc', () => {
 				await uc.create(user.id, failingTaskCardCreateParams);
 			}).rejects.toThrow(ValidationError);
 		});
+		it('should not throw if course end is before due date (same date, different time)', async () => {
```

Comment:
I understand the meaning of the specs in columns, but still I find the naming of the test irritating and hard to understand. I think we should find a better way of determing the test where it should fail (same date) with the one that should pass (different time)

Reply:
agree. Don't know if the naming is better now.

Reply:
improved test naming. Martin and me spoke about it, so I set this comment to resolved